packages/flutter/test/widgets/nested_scroll_view_test.dart

```diff
@@ -566,4 +566,80 @@ void main() {
     expect(buildCount, expectedBuildCount);
     expect(find.byType(NestedScrollView), isNot(paints..shadow()));
   });
+
+  testWidgets('NestedScrollView and iOS bouncing', (WidgetTester tester) async {
```

Comment:
nit: I would add a comment explaining what is special about this sequence of event and what was the bug it's catching

Reply:
added a comment explaining what this tests